rbx_reflection/src/database.rs

```diff
@@ -39,6 +39,24 @@ impl<'a> ReflectionDatabase<'a> {
             enums: HashMap::new(),
         }
     }
+
+    /// Returns a list of superclasses for the provided class name. This list
+    /// will start with the provided class and end with `Instance` if the class
+    /// exists.
+    pub fn superclasses(&self, class_name: &str) -> Option<Vec<&ClassDescriptor>> {
```

Comment:
```suggestion
    pub fn superclasses(&self, class_name: &str) -> Vec<&ClassDescriptor> {
```

Did you still want this return to be optional?

Reply:
The other option would be returning an empty vector in case of an invalid class name, which turns `value.is_some()` into `value.is_empty()`.

I think it's fine for it to be optional as a result.